crm_claim_code/models/crm_claim.py

```diff
@@ -29,3 +30,14 @@ def copy(self, default=None):
             default = {}
         default['code'] = self.env['ir.sequence'].get('crm.claim')
         return super(CrmClaim, self).copy(default)
+
+    @api.multi
+    @api.depends('code')
```

Comment:
No, it isn't a new API computed field, which is the one that needs depends. If it was a new computed field, it won't return a value, but assign it. Old style computed fields require triggers.